pkgs/by-name/un/unnamed-sdvx-clone/package.nix

```diff
@@ -0,0 +1,107 @@
+{
+  stdenv,
+  cmake,
+  fetchFromGitHub,
+  freetype,
+  pkg-config,
+  SDL2,
+  libpng,
+  libjpeg,
+  zlib,
+  libogg,
+  libvorbis,
+  libarchive,
+  iconv,
+  openssl,
+  curl,
+  libcpr,
+  rapidjson,
+  writeShellScriptBin,
+  makeDesktopItem,
+  lib,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "unnamed-sdvx-clone";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "Drewol";
+    repo = "unnamed-sdvx-clone";
+    rev = "refs/tags/v${finalAttrs.version}";
+    fetchSubmodules = true;
+    hash = "sha256-wuf7xZztoxzNQJzlJOfH/Dc25/717NevBx7E0RDybho=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    freetype
+    SDL2
+    libpng
+    libjpeg
+    zlib
+    libogg
+    libvorbis
+    libarchive
+    iconv
+    openssl
+    curl
+    libcpr
+    rapidjson
+  ];
+
+  # FIXME: Set new commit hash when updating
+  cmakeFlags = [
+    "-DUSE_SYSTEM_CPR=ON"
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DGIT_COMMIT=260b050569413a04db9d437c5158d2a032873ad6"
```

Comment:
not a requirement, but if you could explore a different way to obtain this value that doesn't involve needing to hardcore it on a per-update basis that'd be really cool. either that, or if you could write a `passthru.updateScript` that handles this requirement automatically. would just leave less surface area for messing up an update to this drv.

Reply:
looks like its pointless since it checks releases instead if GIT_COMMIT isnt defined

Reply:
done